app/assets/javascripts/common/password_validation.js

```diff
@@ -0,0 +1,53 @@
+PasswordValidation = function() {
+  const DebounceTimeout = 500;
```

Comment:
The timeout exists to avoid hitting the back end with every keystroke. 500ms seemed like a reasonable amount of time to wait between keystrokes, but I may change this in the next ticket if it feels too slow or fast.